Notifications/NotificationService.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  NotificationService.swift
+//  Notifications
+//
+//  Created by Ezequiel Becerra on 24/01/2018.
+//  Copyright © 2018 Ezequiel Becerra. All rights reserved.
+//
+
+import UserNotifications
+
+class NotificationService: UNNotificationServiceExtension {
+    
+    var contentHandler: ((UNNotificationContent) -> Void)?
+    var bestAttemptContent: UNMutableNotificationContent?
+    
+    override func didReceive(_ request: UNNotificationRequest,
+                             withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
+        
+        func save(_ identifier: String, data: Data, options: [AnyHashable: Any]?) -> UNNotificationAttachment? {
+            
+            var directory = URL(fileURLWithPath: NSTemporaryDirectory())
+            directory = directory.appendingPathComponent(
+                ProcessInfo.processInfo.globallyUniqueString,
+                isDirectory: true)
+            
+            do {
+                try FileManager.default.createDirectory(at: directory, withIntermediateDirectories: true, attributes: nil)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/2116f8d687ff9cf2ddfe00a13535cdb270d1c54c/Notifications/NotificationService.swift#L27"></span>Line should be 120 characters or less: currently 122 characters</td>
    </tr>
  </tbody>
</table>